tests/support_server/configure.pm

```diff
@@ -50,6 +51,26 @@ sub run {
     script_output($script);
 }
 
+sub _install_packages {
```

Comment:
In case you want to emphasize that method is considered to be private, let's be consistent and either put it everywhere in the module or not add it at all. WDYT?

Reply:
It means, that you put `_` as prefix for all private methods in the module.

Reply:
i missed this in `turnoff_gnome_screensaver_and_suspend`. my bad. i fixed it